mysqlite.lua

```diff
@@ -125,15 +125,12 @@ local function loadMySQLModule()
     moo, tmsql = file.Exists("bin/gmsv_mysqloo_*.dll", "LUA"), file.Exists("bin/gmsv_tmysql4_*.dll", "LUA")
 
     if not moo and not tmsql then
-        error("Could not find a suitable MySQL module. Supported modules are MySQLOO and tmysql4.")
+        error("Could not find a suitable MySQL module. Supported modules are MySQLOO and tmysql4.\n")
```

Comment:
Is that needed? I know `Error` and `ErrorNoHalt` do but I thought `error` did not.